_test/test/serve_integration_test.dart

```diff
@@ -15,7 +21,13 @@ void main() {
   late HttpClient httpClient;
 
   setUpAll(() {
-    httpClient = HttpClient();
+    // Configure the client so it is possible to download a large number
```

Comment:
When you merge please update the commit message to include this change.

Reply:
Will do!